packages/container/lib/container.js

```diff
@@ -182,6 +182,10 @@ Container.prototype = {
   }
 })();
 
+function isSingleton(container, fullName) {
+  return container._registry.getOption(fullName, 'singleton') !== false;
```

Comment:
On second thought, I don't see an upside to being more accepting of truthy / falsy values in settings completely controlled by developers (which should therefore have explicit values like `false`), so I'd recommend keeping it as is.